website/source/docs/builders/googlecompute.html.md

```diff
@@ -109,6 +109,32 @@ is assumed to be the path to the file containing the JSON.
 }
 ```
 
+### Windows Example
+
+Running WinRM requires that it is opened in the firewall and that the VM enables WinRM for the
+user used to connect in a startup-script.
+
+``` {.json}
+{
+  "builders": [{
+    "type": "googlecompute",
+    "account_file": "account.json",
+    "project_id": "my project",
+    "source_image": "windows-server-2016-dc-v20170227",
+    "disk_size": "50",
+    "machine_type": "n1-standard-1",
+    "communicator": "winrm",
+    "winrm_username": "packer_user",
+    "winrm_insecure": true,
+    "winrm_use_ssl": true,
+    "metadata": {
+      "windows-startup-script-cmd": "winrm quickconfig -quiet & net user /add packer_packer & net localgroup administrators packer_user /add & winrm set winrm/config/service/auth @{Basic=\"true\"}"
```

Comment:
**instead of**

"windows-startup-script-cmd": "winrm quickconfig -quiet & net user /add packer_packer & net localgroup administrators packer_user /add & winrm set winrm/config/service/auth @{Basic=\"true\"}"

**it should be**

"windows-startup-script-cmd": "winrm quickconfig -quiet & net user /add **packer_user** & net localgroup administrators packer_user /add & winrm set winrm/config/service/auth @{Basic=\"true\"}"